### PR TITLE
Add test vectors for typeF91A

### PR DESCRIPTION
**Pull request by thibmeu**
Adding test from the [Rust implementation](https://github.com/raphaelrobert/privacypass/tree/main/tests/kat_vectors) for typeF91A.

I've confirmed both pass. One has been generated with this implementation, the other one with the rust implementation.

### CODE REVIEW

**Comment by thibmeu** [comment]
relates to https://github.com/ietf-wg-privacypass/ietf-draft-privacypass-batched-tokens/issues/20

**Review by Lekensteyn — APPROVED** [review]
The typo fix in the environment variable looks reasonable. After that, `make vectors` does produce `tokens/typeF91A/typeF91A-issuance-test-vectors.json` as expected.

I don't have enough knowledge to assess whether the Rust test vectors are correct, but the format seems reasonable.

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,6 @@
 test:
 	go test ./...
+	TYPEF91A_ISSUANCE_TEST_VECTORS_IN=typeF91A-issuance-test-vectors-rust.json go test -v -run TestVectorVerifyBatchedPrivateIssuance ./tokens/typeF91A
```
Comment (armfazh) [inline Makefile:3]: Note that the [CI script](https://github.com/cloudflare/pat-go/blob/main/.github/workflows/go.yml#L25) never calls this Makefile.

**Review by armfazh — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (thibmeu) [reply Makefile:3]: good catch. the action has been updated and is working as expected

**Pull request merged**